.github/workflows/ci-irctest.yml

```diff
@@ -0,0 +1,115 @@
+# This workflow has four tasks:
+#
+# 1. the first builds inspircd (with some optimizations for irctest), and uploads it
+#    to a temporary storage
+# 2. the other three download the binary we just built, and run it through inspircd,
+#    with either Anope, Atheme, or runs service-independent tests
+
+name: "irctest"
```

Comment:
The other workflows don't quote strings unless necessary. Can you update this workflow to match them?